src/style.css

```diff
@@ -1 +1,99 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-size: normal;
```

Comment:
The `font-size` property is not a valid descriptor for `@font-face`. You should remove this line as it does not affect the font loading.